src/include/stir/recon_buildblock/LogcoshPrior.h

```diff
@@ -196,9 +196,24 @@ class LogcoshPrior:  public
     //! Spatially variant penalty penalty image ptr
     shared_ptr<DiscretisedDensity<3,elemT> > kappa_ptr;
 
+    //! The Log(cosh()) function and its approximation
+    /*!
+     Cosh(x) = 0.5(e^x + e^-x) is an exponential function.
+     Makes the approximation that when |x| > 30, e^-|x| << e^|x| and therefore log(cosh(x)) ~= |x| + log(0.5)
```

Comment:
@zeljkozeljko can you add your line on the bound here? https://github.com/UCL/STIR/issues/768#issuecomment-742181217

please add `[ci skip]` in the first line of the commit message

Reply:
I'm not sure I completely understood your message, but I can at least copy that comment here
<img width="375" alt="Screenshot 2020-12-10 at 01 49 40" src="https://user-images.githubusercontent.com/25665703/101766585-e689bc80-3ada-11eb-8d19-ecfa9dd54399.png">
ie the  log(Cosh(x)) = -log(2) + |x| + log(1 + e^(-2|x|)) = -log(2) + |x|+ O(10^(-27)), for |x|>30

Reply:
best to put it in the actual code. you should be able to edit the file on github. admittedly it'll mean creating a fork etc, so possibly more hassle than it's worth. Alternatively @robbietuk or me do it.

Reply:
Done, it was easier for me to do it I think.